update doc, add information on Calimero console and tutorial (#25)
calimero-is-near/docs: docs/getting_started/access_account.md

---

> title: Accessing your Custodian account

Reviewer: ```suggestion
title: Accessing Custodian accounts
```
The terminology here is extremely ungrateful to work with.
Custodian is the entity which holds the keys.
Under security/custodian, we're talking about accounts for which we have keys in the KMS (the vault).

On the other (linguistical) hand, a custodian account would be an account who has keys in custody.
That's not the case here.

So these accounts are the ones for which Calimero is the custodian.
Extremely ungrateful to work with from the language perspective.

According to chat GPT, we could use the term "Custodial accounts".
This conveys that we are in custody of the accounts.
Custodian is wrongly used here.

Reviewer: Naturally, this needs to be changed in the console as well.
https://calimero.atlassian.net/browse/AM-1173

sidebar_position: 4
---
 

> In the custodian account, we only display those for which we (the shard admin) are key custodians. With your custodian account, you can create a key pair, add public keys and top up your tokens.

Reviewer: ```suggestion
In custodial accounts, we only display those accounts for which the Calimero platform is the custodian. With your custodian account, you can create a key pair, add public keys and top up your tokens.
```
The custodian is the Calimero deployment, not the shard admin, nor Calimero as a company.
Custody is in the k8s cluster, in the shard namespace.
